upload/catalog/controller/product/product.php

```diff
@@ -416,6 +416,13 @@ public function index(): void {
 				$data['review_guest'] = false;
 			}
 
+			if ($this->config->get('config_review_on_purchase') && $this->customer->isLogged()) {
+				$data['review_on_purchase'] = $this->model_catalog_product->cutomerHasPurchase($this->customer->getId(),$product_id) > 0 ? true : false;
+			}else{
+				$data['review_on_purchase'] = true;
+			}
+			var_dump($this->config->get('config_review_on_purchase') ,$this->model_catalog_product->cutomerHasPurchase($this->customer->getId(),$product_id) > 0,$data['review_guest'] );
```

Comment:
Is this needed for specific reasons?

Reply:
no actually was for testing I'll delete it.